### PR TITLE
improve throw If Unchecked

### PR DESCRIPTION
**Pull request by lambaaryan011**
#2160 

fixed error:

/DynFields.java:60: error: [ThrowIfUncheckedKnownChecked] throwIfUnchecked(knownCheckedException) is a no-op.
Throwables.throwIfUnchecked(e);

DynFields.java:69: error: [ThrowIfUncheckedKnownChecked] throwIfUnchecked(knownCheckedException) is a no-op.
Throwables.throwIfUnchecked(e);

### CODE REVIEW

**Comment by lambaaryan011** [comment]
@justinmclean please take a look

**Comment by justinmclean** [comment]
We use Spotless (https://github.com/diffplug/spotless) to keep our code formatting consistent as part of our Gradle build. It's best practice to build the code before checking in. You'll need to run "./gradlew : catalogues: catalogue-hive:spotlessApply" to fix the formatting. I've not looked at the issue in detail, but given the simple change, you may have incorrect line endings. We use Unix/macOS LF, not Windows CFLF.

**Comment by justinmclean** [comment]
@lambaaryan011 tell me if you need some help with this.

**Comment by lambaaryan011** [comment]
ok, sure

**Comment by justinmclean** [comment]
Have you run `./gradlew : catalogues: catalogue-hive:spotlessApply`?

File: catalogs/catalog-hive/src/main/java/com/datastrato/gravitino/catalog/hive/dyn/DynFields.java [hunk 1]
```diff
@@ -57,16 +57,14 @@ public T get(Object target) {
       try {
         return (T) field.get(target);
       } catch (IllegalAccessException e) {
-        Throwables.throwIfUnchecked(e);
         throw new RuntimeException(e);
       }
     }
-
+
```
Comment (justinmclean) [inline catalogs/catalog-hive/src/main/java/com/datastrato/gravitino/catalog/hive/dyn/DynFields.java:63]: You need a new line here not whitespace

**Comment by justinmclean** [comment]
@lambaaryan011 Do you still need a hand with this? If so please give me permission to push to your branch.

**Comment by lambaaryan011** [comment]
Yes, please!   sorry for late  because my college exam is going on

**Comment by justinmclean** [comment]
I don't have permission to push to your repository, so here is the patch.
[patch.txt](https://github.com/datastrato/gravitino/files/14272505/patch.txt)

**Comment by justinmclean** [comment]
@lambaaryan011 Do you have time to apply the patch I gave you?

**Comment by lambaaryan011** [comment]
Yeah, I can do that.

**Comment by shaofengshi** [comment]
Close this PR as it has no update for a long time. Please re-open if needed.

**Pull request closed without merging**